spartans.html

```diff
@@ -68,23 +73,26 @@ <h3 class="HeaderText3">Social Classes</h3><br/><br/>
 They farmed their own land, but had to give half of their crops to the Spartans as payment.<br/>
  Helots were beaten once a year and were forced to wear clothing made from animal skins. Helots caught trying to escape were generally killed.<br/><br/>
 </div>
-<h4 class="HeaderText4">What was it like to grow up as a boy in Sparta?</h4><br/><br/>
+<div id="boysinSparta" class="tabcontent">
+<h4 class="HeaderText4">What was it like to grow up as a boy in Sparta?</h4>
 
 Spartan boys were trained to be soldiers from their youth.<br/>
  They were raised by their mothers until the age of seven and then they would enter a military school called the Agoge.<br/>
   At the Agoge the boys were trained how to fight, but also learned how to read and write.<br/>
 The Agoge was a tough school. The boys lived in barracks and were often beaten to make them tough.<br/>
  They were given little to eat in order to get used to what life would be like when they went to war. <br/>
- The boys were encouraged to fight one another. When the boys turned 20 they entered into the Spartan army.<br/><br/>
-
+ The boys were encouraged to fight one another. When the boys turned 20 they entered into the Spartan army.
+</div>
+<div id="girlsinSparta" class="tabcontent">
 <h7 class="HeaderText7">What was it like to grow up as a girl in Sparta?</h7> <br/><br/>
```

Comment:
don't use h7. Use h1, h2, h3 and h4 only.
In your css styling you can target h1 instead of a class name